Overhaul Interactions section (#412)
discord-userdoccers/discord-userdoccers: pages/topics/gateway-events.mdx

| 1     | UNKNOWN                                                      | Unknown                                                   |
| 2     | TIMEOUT                                                      | The interaction timed out                                 |
| 3     | ACTIVITY_LAUNCH_UNKNOWN_APPLICATION                          | Unknown application                                       |
| 4     | ACTIVITY_LAUNCH_UNKNOWN_CHANNEL                              | Unknown channel                                           |
| 5     | ACTIVITY_LAUNCH_UNKNOWN_GUILD                                | Unknown guild                                             |
| 6     | ACTIVITY_LAUNCH_INVALID_PLATFORM                             | Invalid platform                                          |
| 7     | ACTIVITY_LAUNCH_NOT_IN_EXPERIMENT                            | The guild/user is not eligible for an experiment          |
| 8     | ACTIVITY_LAUNCH_INVALID_CHANNEL_TYPE                         | Invalid channel type                                      |
| 9     | ACTIVITY_LAUNCH_INVALID_CHANNEL_NO_AFK                       | Cannot launch activity in an AFK channel                  |

> | 10    | ACTIVITY_LAUNCH_INVALID_DEV_PREVIEW_GUILD_SIZE               | Unknown                                                   |

Reviewer: ```suggestion
| 10    | ACTIVITY_LAUNCH_INVALID_DEV_PREVIEW_GUILD_SIZE               | Guild is too large for the beta test of this feature |
```

| 11    | ACTIVITY_LAUNCH_INVALID_USER_AGE_GATE                        | Unknown                                                   |

> | 12    | ACTIVITY_LAUNCH_INVALID_USER_VERIFICATION_LEVEL              | Unknown                                                   |

Reviewer: ```suggestion
| 12    | ACTIVITY_LAUNCH_INVALID_USER_VERIFICATION_LEVEL              | User does not meet the guild's verification level |
```

> | 13    | ACTIVITY_LAUNCH_INVALID_USER_PERMISSIONS                     | Unknown                                                   |

Reviewer: ```suggestion
| 13    | ACTIVITY_LAUNCH_INVALID_USER_PERMISSIONS                     | User has insufficient permissions for this interaction |
```

| 14    | ACTIVITY_LAUNCH_INVALID_CONFIGURATION_NOT_EMBEDDED           | The application is not an embedded activity               |

> | 15    | ACTIVITY_LAUNCH_INVALID_CONFIGURATION_PLATFORM_NOT_SUPPORTED | The embedded activity does not support current platform   |

Reviewer: ```suggestion
| 15    | ACTIVITY_LAUNCH_INVALID_CONFIGURATION_PLATFORM_NOT_SUPPORTED | The embedded activity does not support the current platform   |
```

> | 16    | ACTIVITY_LAUNCH_INVALID_CONFIGURATION_PLATFORM_NOT_RELEASED  | The embedded activity does not support current platform   |

Reviewer: ```suggestion
| 16    | ACTIVITY_LAUNCH_INVALID_CONFIGURATION_PLATFORM_NOT_RELEASED  | The embedded activity does is not released for the current platform   |
```

| 17    | ACTIVITY_LAUNCH_FAILED_TO_LAUNCH                             | Failed to launch the activity                             |
| 18    | ACTIVITY_LAUNCH_INVALID_USER_NO_ACCESS_TO_ACTIVITY           | The user does not have permissions to launch the activity |
| 19    | ACTIVITY_LAUNCH_INVALID_LOCATION_TYPE                        | Failed to launch the activity                             |

> | 20    | ACTIVITY_LAUNCH_INVALID_USER_REGION_FOR_APPLICATION          | The embedded activity is not supported in current region  |

Reviewer: ```suggestion
| 20    | ACTIVITY_LAUNCH_INVALID_USER_REGION_FOR_APPLICATION          | The embedded activity is not supported in the current region  |
```

#### Interaction Success

> Sent when interaction succeeds.

Reviewer: ```suggestion
Sent when an interaction succeeds.
```

###### Interaction Success Structure

| Field | Type      | Description                                                 |
| ----- | --------- | ----------------------------------------------------------- |
| id    | snowflake | The ID of the interaction                                   |
| nonce | ?string   | The interaction's nonce, used for interaction deduplication |

#### Application Command Autocomplete Response

> Sent when application responds to autocomplete interaction.

Reviewer: ```suggestion
Sent when an application responds to an autocomplete interaction.
```

###### Application Command Autocomplete Response Structure

| Field   | Type                                                                                                                              | Description                                                 |
| ------- | --------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------- |

> | choices | array[[application command option choice](/interactions/application-commands#application-command-option-choice-structure) object] | The choices the application is autocompleting option with   |

Reviewer: ```suggestion
| choices | array[[application command option choice](/interactions/application-commands#application-command-option-choice-structure) object] | The choices the application is offering for autocomplete |
```

| nonce   | ?string                                                                                                                           | The interaction's nonce, used for interaction deduplication |

#### Interaction Modal Create

> Sent when application responds to interaction with a modal.

Reviewer: ```suggestion
Sent when an application responds to interaction with a modal.
```
